packagebuild/daisy_startupscript_rpm.sh

```diff
@@ -94,6 +96,20 @@ for dir in ${RPMDIR}/{SOURCES,SPECS}; do
   [[ -d "$dir" ]] || mkdir -p "$dir"
 done
 
+EXTRA_RPMBUILD_FLAGS=""
+if [[ -n "EXTRA_REPO" ]]; then
+  EXTRA_RPMBULD_FLAGS="--define 'has_extra_source 1'"
+
+  pushd $BUILD_DIR
```

Comment:
I don't think we set BUILD_DIR

Reply:
indeed. even if we were it would be wrong, I sent a new version doing it right. Thank you.